test/resolver.test.coffee

```diff
@@ -18,17 +18,17 @@ describe "Resolver", ->
       assert.equal this.r.satisfy("0.8.15"), "0.8.15"
 
     it "matches common patterns to stable version", ->
-      assert.equal this.r.satisfy("0.10.x"), this.r.getLatestStable()
-      assert.equal this.r.satisfy("~0.10.0"), this.r.getLatestStable()
+      assert.equal this.r.satisfy("4.1.x"), this.r.getLatestStable()
+      assert.equal this.r.satisfy("~4.1.0"), this.r.getLatestStable()
```

Comment:
Matching rules for unstable versions changed in newer semver, see the commit description.